api/server/server.go

```diff
@@ -49,15 +50,17 @@ type ServerConfig struct {
 	TLSConfig   *tls.Config
 }
 
+// Server contains instance details for the server
 type Server struct {
 	daemon  *daemon.Daemon
-	cfg     *ServerConfig
+	cfg     *APIServerConfig
 	router  *mux.Router
 	start   chan struct{}
 	servers []serverCloser
 }
 
-func New(cfg *ServerConfig) *Server {
+// New returns new instance of the server
```

Comment:
"New returns a new instance of the server based on the specified configuration".